coordinator/migrations/2024-04-03-105900_add_coordinator_liquidation_price/up.sql

```diff
@@ -2,4 +2,5 @@ ALTER TABLE "positions"
     ADD COLUMN coordinator_liquidation_price REAL NOT NULL DEFAULT 0;
 
 UPDATE positions SET coordinator_liquidation_price = average_entry_price * coordinator_leverage / (coordinator_leverage + 1) where trader_direction='short';
-UPDATE positions SET coordinator_liquidation_price = average_entry_price * coordinator_leverage / (coordinator_leverage - 1) where trader_direction='long';
+UPDATE positions SET coordinator_liquidation_price = average_entry_price * coordinator_leverage / (coordinator_leverage - 1) where trader_direction='long' and coordinator_leverage > 1;
+UPDATE positions SET coordinator_liquidation_price = average_entry_price * coordinator_leverage where trader_direction='long' and coordinator_leverage = 1;
```

Comment:
Imho, based on this formula: 

```
(price_opening*leverage)/((leverage+1-(maintenance_margin*leverage)))
```

it should look like this: 

```suggestion
UPDATE positions SET coordinator_liquidation_price = average_entry_price * coordinator_leverage / (coordinator_leverage + 1) where trader_direction='long';

```

Reply:
whoops I've overlooked the direction.. we need the short direction as well 🙈

Reply:
ah, things are so confusing... don't mind me then 🙈